src/Pmi/Entities/AuditLog.php

```diff
@@ -1,19 +1,12 @@
 <?php
 namespace Pmi\Entities;
 
-use GDS\Schema;
 use Pmi\Datastore\Entity;
 
 class AuditLog extends Entity
 {
-    protected static function buildSchema()
+    protected static function getKind()
     {
-        return (new Schema('AuditLog'))
-            ->addString('action')
-            ->addString('user')
-            ->addString('site')
-            ->addDatetime('timestamp')
```

Comment:
If we no longer need explicitly defined schemas, how are data types handled? Is it just implicit from the data passed in for that value (i.e. by passing in a DateTime object, it automatically sets that field as a timestamp?)